test/Riok.Mapperly.Tests/TestHelper.cs

```diff
@@ -7,10 +7,15 @@ namespace Riok.Mapperly.Tests;
 
 public static class TestHelper
 {
-    public static Task VerifyGenerator(string source, TestHelperOptions? options = null)
+    public static Task<VerifyResult> VerifyGenerator(string source, TestHelperOptions? options = null, params object?[] args)
     {
         var driver = Generate(source, options);
-        return Verify(driver).ToTask();
+        var verify = Verify(driver);
+
+        if (args.Any())
+            verify.UseParameters(args);
```

Comment:
This code base uses braces for ifs except if it is a simple return/throw statement.

Reply:
Fixed.